src/jsp/com/cubrid/jsp/Server.java

```diff
@@ -86,6 +87,12 @@ private Server(
             loggingThread = new LoggingThread(logFilePath);
             loggingThread.start();
 
+            tmpPath = System.getProperty("CUBRID_TMP");
+            if (tmpPath == null) {
+                tmpPath = rootPath + File.separator + "tmp";
```

Comment:
If CUBRID_TMP is null, is tmpPath $CUBRID/tmp ?
How about changing default tmpPath to $CUBRID/var/CUBRID_SOCK ?

Reply:
java.io.tmpdir path is not only for socket file, but Any file that a user create by createTmpFile() API in Java SP.

Reply:
OK, i got it.